Woof/Woof/Helper/Color+Extension.swift

```diff
@@ -0,0 +1,10 @@
+import SwiftUI
+
+/// Describes the color scheme using in the application
+extension Color {
```

Comment:
Imagine I use the colour in the app:
```swift
let bgColour = Color.grayLight
```

How can I understand this is a part of app color scheme?

I'd suggest to have it like this:
```swift
let bgColour = AppColor.grayLight
```

or

```swift
let bgColour = Color.App.grayLight
```

Does that make sense?